mock-data/MockRiddles.ts

```diff
@@ -1,24 +1,67 @@
-import type { CountryCode } from '../src/utils/CountryCodes';
-import type { Difficulty } from '../src/utils/Difficulty';
 import { getDifficultyObject } from '../src/utils/Difficulty';
 import { RiddleStatus } from '../src/utils/Enums';
+import type {
+	QuestionDisplayDetail,
+	QuestionUpsertDetail,
+	RiddleDisplayDetail,
+	RiddlePreview,
+	RiddleUpsertDetail
+} from '../src/utils/Types.ts';
 
-//TODO: Move type declaration to types
+const MockUpsertQuestions: QuestionUpsertDetail[] = [
+	{
+		id: 1,
+		number: 2,
+		text: 'It is blue and annoying, what is it?',
+		image: 'public/vite.svg',
+		hints: [],
+		correctAnswers: [{ text: 'Smurf' }]
+	},
+	{
+		id: 2,
+		number: 1,
+		text:
+			'    Ten centuries shall the fortress stand\n' +
+			'    Walls of spirit wrapped in walls of fire\n' +
+			'    And horned lords shall bowl their head\n' +
+			'    To one not yet born, of the darkest sire\n' +
+			'\n' +
+			'    One century of blood and strife\n' +
+			'    The moon shall darken, and none know why\n' +
+			'    The resting place at last is found\n' +
+			'    Of the Seventh, who soared so high\n' +
+			'\n' +
+			'    Last daughter of a forlorn line\n' +
+			'    Shall guide him into history\n' +
+			'    Beneath the crypts prophecies clash\n' +
+			'    The war of ancient enemies',
+		hints: [
+			{ text: 'Heroes', order: 1 },
+			{ text: 'Might & Magic', order: 2 },
+			{ text: '5. installment', order: 3 }
+		],
+		correctAnswers: [{ text: 'Smurf' }]
```

Comment:
Nemohlo by to být pouze pole stringů? Pokud bude pouze jediná položka v objektu (`text`), tak za mě nedává tolik smysl.

Reply:
Spíš ne, kvůli updatu - když nemáš idčko, neuděláš update

Reply:
Změněno na string reprezentující pole, bude aktualizováno dle db